### PR TITLE
Lazily load (most) images

### PR DESCRIPTION
**Pull request by plural**
This is another improvement suggested by the lighthouse audit.  This should help out page responsiveness.

### CODE REVIEW

**Comment by NoahTheDuke** [comment]
This is very cool. Won’t have much of an effect until the browsers support it (only Firefox at the moment), but will be great after that happens.

**Comment by plural** [comment]
hmm, i'm not seeing what isn't supported by firefox here, but i'll check on firefox directly in a bit.

**Comment by NoahTheDuke** [comment]
My apologies, my understanding of the support was wrong (and I phrased it backwards lol). Looks like the modern browsers except Safari support it: [here](https://www.caniuse.com/#search=loading)

**Comment by plural** [comment]
gotcha.  I'll double-check on a safari browser in a bit, as well.  I think
this lib is just using JS for it and not using that attribute, so we should
be good, but i will verify.

-jason

File: app/Resources/views/Decklist/decklist-row-always-xs.html.twig.orig [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<<<<<<< HEAD
```
Comment (NoahTheDuke) [inline app/Resources/views/Decklist/decklist-row-always-xs.html.twig.orig]: uh oh, looks like a merge conflict snuck in

Comment (plural) [reply app/Resources/views/Decklist/decklist-row-always-xs.html.twig.orig]: oops!  Looks like i let 2 files accidentally get checked in from a merge conflict.

**Comment by plural** [comment]
thanks for taking another look, Noah.

File: app/Resources/views/Decklist/decklist-row-always-xs.html.twig [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <tr>
   <td style="width:67px">
-    <img src="{{ decklist.identity_url ? decklist.identity_url : asset('/card_image/'~decklist.code~'.png') }}" alt="{{ decklist.identity }}" style="width:51px;height:71px">
+    <img data-src="{{ decklist.identity_url ? decklist.identity_url : asset('/card_image/'~decklist.code~'.png') }}" alt="{{ decklist.identity }}" style="width:51px;height:71px" class="lazyload">
```
Comment (NoahTheDuke) [inline app/Resources/views/Decklist/decklist-row-always-xs.html.twig:3]: `src` is a [required attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img) of `img` tags. Did you mean to add an additional `data-src` tag?

Comment (plural) [reply app/Resources/views/Decklist/decklist-row-always-xs.html.twig:3]: no, the way this works is by taking advantage of the fact that a missing src is basically a no-op.  With a src, the browser makes the request immediately.  With data-src, the js will do it when appropriate.

**Pull request merged**